tests/unittests/BondSplicerTests.cpp

```diff
@@ -32,7 +32,7 @@ class ShadowBondSplicer : protected MAT::BondSplicer
     {
         FullDumpBinaryBlob output;
         static_cast<std::vector<uint8_t>&>(output) = MAT::BondSplicer::splice();
-        return std::move(output);
```

Comment:
The compiler flagged this as an error, saying that it is a redundent call to `std::move`

Reply:
Probably pessimizing move. Good fix.

Reply:
Seems return value optimization does apply here so explicit move is not required.